app/components/InfoList/index.tsx

```diff
@@ -15,21 +17,23 @@ type Props = {
  * Location <strong>Oslo</strong>
  * Time <strong>Yesterday</strong>
  */
-function InfoList({ items, className }: Props) {
+const InfoList = ({ items, className }: Props) => {
   return (
-    <table className={className}>
-      <tbody>
-        {items.filter(Boolean).map(({ key, keyNode, value }) => (
-          <tr key={key}>
-            <td>{keyNode ?? <span>{key}</span>}</td>
-            <td>
-              <strong>{value}</strong>
-            </td>
-          </tr>
-        ))}
-      </tbody>
-    </table>
+    <Flex column gap="0.5rem" className={className}>
+      {items.filter(isNotNullish).map(({ key, keyNode, value }) => (
+        <Flex
+          alignItems="center"
+          justifyContent="space-between"
+          gap="1rem"
+          key={key}
+        >
+          {keyNode ?? <span>{key}</span>}
+
+          <span style={{ fontWeight: '600', textAlign: 'right' }}>{value}</span>
```

Comment:
could we pop this in to styles?

Reply:
There was no css file so I just didn't bother, but I moved it now 👍🏼